### ISSUE TITLE
panic running "cloudquery query"

### ISSUE THREAD

**Issue by HG00** [cloudquery/cloudquery#65]
#### OS
Distributor ID: Ubuntu
Description:    Ubuntu 20.04 LTS
Release:        20.04

#### version
Version: 0.9.6
Commit: 01029208a35918972e3cdec653c5a8e7557717dd
Date: 2021-02-26 17:19:16.512656639 +0000 UTC m=+0.028906351

#### To reproduce

Running the setup per readme, i can fetch successfully and populate a local sqlite db. Generating policy for aws_cis, when I run cloudquery query, i get a panic:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xcd4e3c]

goroutine 1 [running]:
github.com/cloudquery/cloudquery/cloudqueryclient.(*Client).RunQuery(0xc000422300, 0x107c55a, 0xc, 0x0, 0x0, 0xc000422300, 0x0)
        /home/ubuntu/sauce/cloudquery/cloudqueryclient/client.go:231 +0x6c
github.com/cloudquery/cloudquery/cmd.glob..func7(0x19601a0, 0x19f9c58, 0x0, 0x0, 0x0, 0x0)
        /home/ubuntu/sauce/cloudquery/cmd/query.go:24 +0x316
github.com/spf13/cobra.(*Command).execute(0x19601a0, 0x19f9c58, 0x0, 0x0, 0x19601a0, 0x19f9c58)
        /home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0x1960440, 0x1086c18, 0x18, 0x0)
        /home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/cloudquery/cloudquery/cmd.Execute()
        /home/ubuntu/sauce/cloudquery/cmd/root.go:24 +0x2d
main.main()
        /home/ubuntu/sauce/cloudquery/main.go:14 +0x82
```

Trying with postgres i get the same error. Is there something I've missed?

**Comment by yevgenypats** [comment]
@HG00 thanks for reporting! Just pushed a fix to [v0.9.7](https://github.com/cloudquery/cloudquery/releases/tag/v0.9.7). You can try again with the SQLite, it should work.

**Comment by HG00** [comment]
Perfect, works with sqlite thanks @yevgenypats - tested with Version: 0.9.7 and works as expected
